content/en/blog/_posts/2023-11-20-volume-attributes-class/index.md

```diff
@@ -0,0 +1,162 @@
+# Kubernetes v1.29: VolumeAttributesClass for Volume Modification
```

Comment:
[See my comment](https://github.com/kubernetes/website/pull/43777#issuecomment-1815547950) about the correct formatting for the blog post

Reply:
Updated base on comments: https://github.com/kubernetes/website/pull/43777/commits/3367a8769727355562693894f777af77b7cf592b